Dockerfile

```diff
@@ -57,6 +54,7 @@ RUN apt-get update && \
         mysql-client \
         poppler-utils \
         sudo \
+        vim \
```

Comment:
BTW, one more thing: if you are looking at amending the development setup after the recent Invenio package split, then please see my previous comments about possibly homogenising `/code/invenio` and `/code/src/*`:

- https://github.com/inveniosoftware/invenio/pull/3380#issuecomment-138509850

E.g. my first comment there seems to touch your file ownership vs sudo righs issue.

One possible option coud be to have a development setup where people will check out on their host systems all the independent packages:

```
src/invenio-records
src/invenio-documents
```

and we could directly mount them as docker volumes inside the container, and depend on `../invenio-foo/invenio_foo/` like package paths for easy editing.

P.S. See also some previous debug/devel/prod docker setup musings at:

- https://github.com/inveniosoftware/invenio/issues/3286